src/App.vue

```diff
@@ -1,25 +1,33 @@
 <template>
   <div
-    v-if="config"
     id="app"
+    v-if="config"
     :class="[
-      `theme-${config.theme}`,
-      `page-${currentPage}`,
-      isDark ? 'dark' : 'light',
+      'theme-' + config.theme,
+      'page-' + currentPage,
```

Comment:
This is not the recommended way to concatenate string, the proper way to do it is the way it is. Could you please revert this?